### ISSUE TITLE
Can't delete because folder path is too long.

### ISSUE THREAD

**Issue by msguerra74** [gruntjs/grunt-contrib-imagemin#209]
I don't know if it's just me or not, but I'm having a really hard time deleting the grunt-contrib-imagemin folder from inside node_modules. I'm running Windows 8.1 (64-bit) and it tells me the path is too long. I've looked at the folder  structure and it gets really nested down so deeply that I have to really jump through hoops to finally get it all deleted. This just started happening sometime in the last 2 or 3 updates, so I was wondering if anyone else was having this issue? Thanks

**Comment by Sabbir786** [comment]
anyone used  “Long Path Tool”
This tool is great 
Thanks

**Comment by aamarks** [comment]
I've just figured this out and don't know a lot about this, but the newest version(s) of npm handles installation by flattening the directory structure of dependencies and thus installs grunt-contrib-imagemin without exceeding windows file length limits. To "fix" old installs first uninstall problematic plugins such as this one and grunt-contrib-uglify, and then reinstall. Simply updating plugins won't flatten. (My current installed versions are Nodejs v.4.5. which includes npm 2.15. which I then upgraded to npm 3.10.)
